config/sprayproxy.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright © 2023 The Spray Proxy Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sprayproxy
+  namespace: redhat-sprayproxy
+  labels:
+    app.kubernetes.io/name: sprayproxy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: sprayproxy
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: sprayproxy
+    spec:
+      serviceAccountName: redhat-sprayproxy
+      containers:
+        - name: sprayproxy
+          image: ko://github.com/redhat-appstudio/sprayproxy
+          args:
+            - server
+          env:
+            - name: SPRAYPROXY_SERVER_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: sprayproxy-config
+                  key: HOST
+                  optional: true
+            - name: SPRAYPROXY_SERVER_PORT
+              valueFrom:
+                secretKeyRef:
+                  name: sprayproxy-config
+                  key: PORT
+                  optional: true
+            - name: SPRAYPROXY_SERVER_BACKENDS
+              valueFrom:
+                secretKeyRef:
+                  name: sprayproxy-config
+                  key: BACKENDS
+                  optional: false
+            - name: SPRAYPROXY_SERVER_INSECURE_SKIP_TLS_VERIFY
+              valueFrom:
+                secretKeyRef:
+                  name: sprayproxy-config
+                  key: INSECURE
+                  optional: true
```

Comment:
Similarly this can be removed. This should only be set in an overlay patch for development.